chapter_recurrent-neural-networks/sequence_vn.md

```diff
@@ -144,7 +144,13 @@ Statisticians call dynamics that do not change *stationary*.
 Regardless of what we do, we will thus get an estimate of the entire time series via
 -->
 
-*dịch đoạn phía trên*
+Cả hai trường hợp đều đặt ra câu hỏi về cách tạo ra dữ liệu huấn luyện.
+Người ta thường sử dụng các quan sát quá khứ cho đến hiện tại để dự đoán các quan sát xảy ra trong tương lai.
+Chúng ta không hi vọng thời gian sẽ đứng yên.
+Tuy nhiên, một giả định phổ biến là trong khi các giá trị cụ thể của $x_t$ có thể thay đổi, thì ít nhất động lực của chuỗi thời gian sẽ không đổi.
```

Comment:
`dynamic` chỗ này nghĩa là `a pattern or process of change, growth, or activity` trong kiểu `population dynamics`.
Mình ko biết từ `động lực` trong Tiếng việt có hiểu theo nghĩa này được ko nhỉ hay nên dịch thẳng ra.

Reply:
hm em lại hiểu `dynamic` chỗ này là `thứ tạo ra quá trình thay đổi theo thời gian`, gần gần kiểu `a force that stimulates a pattern or process of change, growth, or activity`.